### PR TITLE
[Doc Improvement][Update the docs set with consistent app manifest terminology][3124756] 

### PR DESCRIPTION
**Pull request by v-ganr**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/tab-sso-troubleshooting.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6d85bb2c25cfea2b2b9a149c6516ea215f8f466f/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-troubleshooting.md) | 96 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b1f56fb8-d01f-484f-9e22-21e37212641d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/TeamsFx-preview-and-customize-app-manifest.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6d85bb2c25cfea2b2b9a149c6516ea215f8f466f/msteams-platform%2Ftoolkit%2FTeamsFx-preview-and-customize-app-manifest.md) | 93 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/251c93e8-b3e3-492d-a28e-4ad60bd1f9ae) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-mobile.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6d85bb2c25cfea2b2b9a149c6516ea215f8f466f/msteams-platform%2Ftoolkit%2Fdebug-mobile.md) | 96 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fb479a8c-8508-4152-ae6a-745361b21205) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6d85bb2c25cfea2b2b9a149c6516ea215f8f466f/msteams-platform%2Ftoolkit%2Fdebug-overview.md) | 94 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c3531836-e106-4d0a-871e-563414ae7486) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/provision.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6d85bb2c25cfea2b2b9a149c6516ea215f8f466f/msteams-platform%2Ftoolkit%2Fprovision.md) | 92 | 62 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/40f50c44-961f-4f6a-b306-6d0cc7b5783a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/webhooks-and-connectors/how-to/connectors-creating.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6d85bb2c25cfea2b2b9a149c6516ea215f8f466f/msteams-platform%2Fwebhooks-and-connectors%2Fhow-to%2Fconnectors-creating.md) | 96 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/79db185d-b7c7-47e8-9178-979e218a6505) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/tab-sso-troubleshooting.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3fc64e01517534a748f190feb75d0fa8fccebcf4/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-troubleshooting.md) | 96 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3e60440c-acf6-4bad-9987-01b7a0dad46d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/TeamsFx-preview-and-customize-app-manifest.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3fc64e01517534a748f190feb75d0fa8fccebcf4/msteams-platform%2Ftoolkit%2FTeamsFx-preview-and-customize-app-manifest.md) | 93 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/67eba843-4ef9-4d75-9f25-19b71bd15669) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-mobile.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3fc64e01517534a748f190feb75d0fa8fccebcf4/msteams-platform%2Ftoolkit%2Fdebug-mobile.md) | 96 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ba544871-4ed0-4a45-aceb-880c9a356b5c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3fc64e01517534a748f190feb75d0fa8fccebcf4/msteams-platform%2Ftoolkit%2Fdebug-overview.md) | 94 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bbb70381-3cad-4ea6-b6ef-cd3a0f973642) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/provision.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3fc64e01517534a748f190feb75d0fa8fccebcf4/msteams-platform%2Ftoolkit%2Fprovision.md) | 92 | 62 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7794d712-84d0-463e-95a6-590c0519753e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/webhooks-and-connectors/how-to/connectors-creating.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3fc64e01517534a748f190feb75d0fa8fccebcf4/msteams-platform%2Fwebhooks-and-connectors%2Fhow-to%2Fconnectors-creating.md) | 96 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e1c0c484-a673-4daf-9a7a-d15f131dced2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/tab-sso-troubleshooting.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d96f7e7ccd1792ad3e4a1a552caee5508c40c7e1/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-troubleshooting.md) | 96 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c5381b66-ed8e-48b1-9c2f-75d03cb786ba) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/TeamsFx-preview-and-customize-app-manifest.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d96f7e7ccd1792ad3e4a1a552caee5508c40c7e1/msteams-platform%2Ftoolkit%2FTeamsFx-preview-and-customize-app-manifest.md) | 93 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d26a6187-7c5e-41d3-8f88-26deaad2b5ed) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-mobile.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d96f7e7ccd1792ad3e4a1a552caee5508c40c7e1/msteams-platform%2Ftoolkit%2Fdebug-mobile.md) | 96 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/89af6b7c-c8d6-4b77-b557-191d21bd2965) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d96f7e7ccd1792ad3e4a1a552caee5508c40c7e1/msteams-platform%2Ftoolkit%2Fdebug-overview.md) | 94 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b51e977c-3913-4cbe-bd85-2d81bca7c353) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/provision.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d96f7e7ccd1792ad3e4a1a552caee5508c40c7e1/msteams-platform%2Ftoolkit%2Fprovision.md) | 92 | 62 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dc1423d8-0ead-4941-8546-f4b88a36b8fb) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/webhooks-and-connectors/how-to/connectors-creating.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d96f7e7ccd1792ad3e4a1a552caee5508c40c7e1/msteams-platform%2Fwebhooks-and-connectors%2Fhow-to%2Fconnectors-creating.md) | 96 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ef04d12a-5ebf-42c3-8b53-786c07d6aef0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/toolkit/TeamsFx-preview-and-customize-app-manifest.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
 ---
-title: Customize Teams App Manifest in Teams Toolkit
+title: Customize App Manifest in Teams Toolkit
```
Comment (erikadoyle) [inline msteams-platform/toolkit/TeamsFx-preview-and-customize-app-manifest.md:2]: Decapitalize "App Manifest" for consistency with other instances across docset

**Review by erikadoyle — CHANGES_REQUESTED** [review]
Just a couple of minor changes--

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/tab-sso-troubleshooting.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2be469519582964e45b7d679195f557761fcf9e3/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-troubleshooting.md) | 96 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/25b26386-3527-41a7-bd21-1a2bd0b472a4) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/TeamsFx-preview-and-customize-app-manifest.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2be469519582964e45b7d679195f557761fcf9e3/msteams-platform%2Ftoolkit%2FTeamsFx-preview-and-customize-app-manifest.md) | 93 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2e9840b6-f8ae-420d-a30d-bdf63ce888ff) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-mobile.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2be469519582964e45b7d679195f557761fcf9e3/msteams-platform%2Ftoolkit%2Fdebug-mobile.md) | 96 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/867528b4-143d-40f2-9930-1dba45e49410) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2be469519582964e45b7d679195f557761fcf9e3/msteams-platform%2Ftoolkit%2Fdebug-overview.md) | 94 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d5b61ac3-2851-4262-8f51-1d82ba8395e7) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/provision.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2be469519582964e45b7d679195f557761fcf9e3/msteams-platform%2Ftoolkit%2Fprovision.md) | 92 | 62 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bd586c9f-2593-4178-af8a-d8e3087550df) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/webhooks-and-connectors/how-to/connectors-creating.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2be469519582964e45b7d679195f557761fcf9e3/msteams-platform%2Fwebhooks-and-connectors%2Fhow-to%2Fconnectors-creating.md) | 96 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e4e7e083-0b77-4d8d-99c3-c3581aeb6b81) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine.

File: msteams-platform/toolkit/TeamsFx-preview-and-customize-app-manifest.md (hunk 1 above)
Comment (v-ganr) [reply msteams-platform/toolkit/TeamsFx-preview-and-customize-app-manifest.md]: Updated.

File: msteams-platform/toolkit/TeamsFx-preview-and-customize-app-manifest.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
 author: surbhigupta
-description: In this module, learn how to edit, preview, and customize Teams App Manifest in different environment.
+description: In this module, learn how to edit, preview, and customize App Manifest in different environment.
```
[the remark this answers is not in the document]
Comment (v-ganr) [reply msteams-platform/toolkit/TeamsFx-preview-and-customize-app-manifest.md]: Updated.

**Review by erikadoyle — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/tab-sso-troubleshooting.md](https://github.com/MicrosoftDocs/msteams-docs/blob/99720037a3c43bbb3d44ec67ca10a549169f44c0/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-troubleshooting.md) | 96 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e36a29e9-0dc9-46e5-98ba-a62d0e76b995) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/TeamsFx-preview-and-customize-app-manifest.md](https://github.com/MicrosoftDocs/msteams-docs/blob/99720037a3c43bbb3d44ec67ca10a549169f44c0/msteams-platform%2Ftoolkit%2FTeamsFx-preview-and-customize-app-manifest.md) | 93 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0cf5279a-444f-4ffb-ae0b-656217df1f3b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-mobile.md](https://github.com/MicrosoftDocs/msteams-docs/blob/99720037a3c43bbb3d44ec67ca10a549169f44c0/msteams-platform%2Ftoolkit%2Fdebug-mobile.md) | 96 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c3ef2507-3b19-4ae0-9a77-13838c3726ea) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/99720037a3c43bbb3d44ec67ca10a549169f44c0/msteams-platform%2Ftoolkit%2Fdebug-overview.md) | 94 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1a539faf-7f19-4ba2-a1f8-304fa1156b31) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/provision.md](https://github.com/MicrosoftDocs/msteams-docs/blob/99720037a3c43bbb3d44ec67ca10a549169f44c0/msteams-platform%2Ftoolkit%2Fprovision.md) | 92 | 62 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4bdff073-b4d4-47f8-b30a-9ff9dfee46d9) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/webhooks-and-connectors/how-to/connectors-creating.md](https://github.com/MicrosoftDocs/msteams-docs/blob/99720037a3c43bbb3d44ec67ca10a549169f44c0/msteams-platform%2Fwebhooks-and-connectors%2Fhow-to%2Fconnectors-creating.md) | 96 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6550a950-50a2-425a-adc3-01a6def22360) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/tab-sso-troubleshooting.md](https://github.com/MicrosoftDocs/msteams-docs/blob/137b97fde6abb9fce89acfd667e2da4a45fb1e63/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-troubleshooting.md) | 96 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fe294c8f-7ca4-41c9-9902-9e84681abdb1) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/TeamsFx-preview-and-customize-app-manifest.md](https://github.com/MicrosoftDocs/msteams-docs/blob/137b97fde6abb9fce89acfd667e2da4a45fb1e63/msteams-platform%2Ftoolkit%2FTeamsFx-preview-and-customize-app-manifest.md) | 92 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0021c872-fa03-43be-b9d4-e024478c8a56) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-mobile.md](https://github.com/MicrosoftDocs/msteams-docs/blob/137b97fde6abb9fce89acfd667e2da4a45fb1e63/msteams-platform%2Ftoolkit%2Fdebug-mobile.md) | 96 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ae387def-af81-4cee-b91c-04d979aa3d44) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/137b97fde6abb9fce89acfd667e2da4a45fb1e63/msteams-platform%2Ftoolkit%2Fdebug-overview.md) | 94 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/adf338a7-782c-48d6-8e81-42b36c801efc) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/provision.md](https://github.com/MicrosoftDocs/msteams-docs/blob/137b97fde6abb9fce89acfd667e2da4a45fb1e63/msteams-platform%2Ftoolkit%2Fprovision.md) | 92 | 62 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9888995a-d9b5-4bb5-ac8e-e386f170805d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/webhooks-and-connectors/how-to/connectors-creating.md](https://github.com/MicrosoftDocs/msteams-docs/blob/137b97fde6abb9fce89acfd667e2da4a45fb1e63/msteams-platform%2Fwebhooks-and-connectors%2Fhow-to%2Fconnectors-creating.md) | 96 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2e66dea4-5d1b-4559-b366-4a0b59789fbe) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/get-started/glossary.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d9218fa17d88804053e12db3b3c413d522450cce/msteams-platform%2Fget-started%2Fglossary.md) | 95 | 40 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/392dc9fc-a3cc-460c-a776-d11ab0a7bf55) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/authentication/tab-sso-troubleshooting.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d9218fa17d88804053e12db3b3c413d522450cce/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-troubleshooting.md) | 96 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3bed89a6-aa2d-4441-9185-5170f4e2b630) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/TeamsFx-preview-and-customize-app-manifest.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d9218fa17d88804053e12db3b3c413d522450cce/msteams-platform%2Ftoolkit%2FTeamsFx-preview-and-customize-app-manifest.md) | 92 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/64485641-b9e7-452a-b16e-845423fd3a21) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-mobile.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d9218fa17d88804053e12db3b3c413d522450cce/msteams-platform%2Ftoolkit%2Fdebug-mobile.md) | 96 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/45c6c145-c2e7-4860-b552-75746a0b37eb) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d9218fa17d88804053e12db3b3c413d522450cce/msteams-platform%2Ftoolkit%2Fdebug-overview.md) | 94 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3a19eb9a-fea6-45ec-b384-269afa3a5915) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/provision.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d9218fa17d88804053e12db3b3c413d522450cce/msteams-platform%2Ftoolkit%2Fprovision.md) | 92 | 62 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f084d3b9-fb35-4ce1-a015-be2bf6cef6d7) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/webhooks-and-connectors/how-to/connectors-creating.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d9218fa17d88804053e12db3b3c413d522450cce/msteams-platform%2Fwebhooks-and-connectors%2Fhow-to%2Fconnectors-creating.md) | 96 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8900a534-02fa-40d0-9715-a3581bd61b1f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/get-started/glossary.md](https://github.com/MicrosoftDocs/msteams-docs/blob/eb0fe95a5e2d5ce8e47dbcf35d9630d335247f4b/msteams-platform%2Fget-started%2Fglossary.md) | 95 | 61 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0693b1f4-9026-4764-96d9-db6597f239c1) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/authentication/tab-sso-troubleshooting.md](https://github.com/MicrosoftDocs/msteams-docs/blob/eb0fe95a5e2d5ce8e47dbcf35d9630d335247f4b/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-troubleshooting.md) | 96 | 26 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/469cb197-2994-413e-b450-d662a25e5572) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/TeamsFx-preview-and-customize-app-manifest.md](https://github.com/MicrosoftDocs/msteams-docs/blob/eb0fe95a5e2d5ce8e47dbcf35d9630d335247f4b/msteams-platform%2Ftoolkit%2FTeamsFx-preview-and-customize-app-manifest.md) | 93 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9c7da2e1-f99e-4ee9-bac1-4a49af7ed326) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-mobile.md](https://github.com/MicrosoftDocs/msteams-docs/blob/eb0fe95a5e2d5ce8e47dbcf35d9630d335247f4b/msteams-platform%2Ftoolkit%2Fdebug-mobile.md) | 96 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c19f40a2-77ee-4e6d-a1ab-cc77ce7e2526) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/eb0fe95a5e2d5ce8e47dbcf35d9630d335247f4b/msteams-platform%2Ftoolkit%2Fdebug-overview.md) | 94 | 19 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/18bb94ac-ca82-43d2-bd34-2b0e832ea7b7) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/provision.md](https://github.com/MicrosoftDocs/msteams-docs/blob/eb0fe95a5e2d5ce8e47dbcf35d9630d335247f4b/msteams-platform%2Ftoolkit%2Fprovision.md) | 92 | 115 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3cb09264-b630-47cb-aef1-8ac3c47cc61d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/webhooks-and-connectors/how-to/connectors-creating.md](https://github.com/MicrosoftDocs/msteams-docs/blob/eb0fe95a5e2d5ce8e47dbcf35d9630d335247f4b/msteams-platform%2Fwebhooks-and-connectors%2Fhow-to%2Fconnectors-creating.md) | 96 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/91b87e15-5f35-4001-8be2-7ea6c33f649f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**